mm-mac2ipv4.tests

```diff
@@ -40,24 +40,38 @@ rand0to255() {
     return $randdec
 }
 
+rand0to255eo32() {
+    ((randdec=$RANDOM / 128))
+    ((randdec=randdec - (randdec % 64 - randdec % 32)))
+    return $randdec
+}
+
 randIPMA() {
     local a
     randMA=("DC:9F:DB" "DC:9F:DB" "DC:9F:DB" "DC:9F:DB")
+    # 1st IPv4 octet
     randIP="100"
-    rand64to127
+    # 2nd IPv4 octet
+    rand96to127
     randMA[0]="${randMA[0]}:$(printf "%02X\n" $((randdec - 64)))"
     randMA[1]="${randMA[1]}:$(printf "%02X\n" $randdec)"
     randMA[2]="${randMA[2]}:$(printf "%02X\n" $((randdec + 64)))"
     randMA[3]="${randMA[3]}:$(printf "%02X\n" $((randdec + 128)))"
     randIP="$randIP.$randdec"
-    for face in ^ ^; do
-        rand0to255
-        randMA[0]="${randMA[0]}:$(printf "%02X\n" $randdec)"
-        randMA[1]="${randMA[1]}:$(printf "%02X\n" $randdec)"
-        randMA[2]="${randMA[2]}:$(printf "%02X\n" $randdec)"
-        randMA[3]="${randMA[3]}:$(printf "%02X\n" $randdec)"
-        randIP="$randIP.$randdec"
-    done
+    # 3rd IPv4 octet
+    rand0to255
+    randMA[0]="${randMA[0]}:$(printf "%02X\n" $randdec)"
+    randMA[1]="${randMA[1]}:$(printf "%02X\n" $randdec)"
+    randMA[2]="${randMA[2]}:$(printf "%02X\n" $randdec)"
+    randMA[3]="${randMA[3]}:$(printf "%02X\n" $randdec)"
+    randIP="$randIP.$randdec"
+    # 4th IPv4 octet
+    rand0to255eo32 # every other 32
```

Comment:
If you have to write a comment about a function name, you probably could have a better function name.